src/backend/Dot.ml

```diff
@@ -80,9 +79,12 @@ module Make(S : Msat.S)(A : Arg with type atom := S.atom
 
   let print_edges fmt n =
     match P.(n.step) with
-    | P.Resolution (p1, p2, _) ->
-      print_edge fmt (res_node_id n) (proof_id p1);
-      print_edge fmt (res_node_id n) (proof_id p2)
+    | P.Hyper_res {P.hr_steps=[];_} -> () (* NOTE: should never happen *)
```

Comment:
If this case isn't supposed to happen, better use an `assert false` to ensure it really doesn't happen ?